libraries/abstractions/common_io/include/iot_i2c.h

```diff
@@ -42,70 +42,69 @@
+#define IOT_I2C_BUSY                      ( 2 )     /*!< I2C bus is busy at current time. */
+#define IOT_I2C_WRITE_FAILED              ( 3 )     /*!< I2C driver returns error when performing write operation. */
+#define IOT_I2C_READ_FAILED               ( 4 )     /*!< I2C driver returns error when performing read operation. */
+#define IOT_I2C_NACK                      ( 5 )     /*!< Unexpected NACK is caught. */
+#define IOT_I2C_BUS_TIMEOUT               ( 6 )     /*!< I2C operation not completed within specified timeout. */
+#define IOT_I2C_NOTHING_TO_CANCEL         ( 7 )     /*!< No ongoing transaction when cancel operation is performed. */
+#define IOT_I2C_FUNCTION_NOT_SUPPORTED    ( 8 )     /*!< I2C operation is not supported. */
+#define IOT_I2C_SLAVE_ADDRESS_NOT_SET     ( 9 )     /*!< Slave address is not set before calling I2C read or write operation. */
 
 /**
  * @brief I2C Bus status
  */
 typedef enum
 {
-    eI2CBusIdle = 0,
-    eI2cBusBusy = IOT_I2C_BUSY,
+    eI2CBusIdle = 0,            /*!< I2C bus is idle. */
+    eI2cBusBusy = IOT_I2C_BUSY, /*!< I2C bus is busy. */
 } IotI2CBusStatus_t;
 
 /**
  * @brief I2C operation status.
  */
 typedef enum
 {
-    eI2CCompleted = IOT_I2C_SUCCESS,
-    eI2CDriverFailed,
-    eI2CNackFromSlave = IOT_I2C_NACK,
-    eI2CMasterTimeout = IOT_I2C_BUS_TIMEOUT,
+    eI2CCompleted = IOT_I2C_SUCCESS,         /*!< I2C operation completed successfully. */
+    eI2CDriverFailed,                        /*!< I2C driver returns error during last operation. */
+    eI2CNackFromSlave = IOT_I2C_NACK,        /*!< Unexpected NACK is caught. */
+    eI2CMasterTimeout = IOT_I2C_BUS_TIMEOUT, /*!< I2C operation not completed within specified timeout. */
 } IotI2COperationStatus_t;
 
 /**
  * @brief I2C bus configuration
  */
 typedef struct IotI2CConfig
 {
-    uint32_t ulMasterTimeout;   /**<! Master timeout value in msec, to relinquish the bus if slave does not respond. This is an extended feature of SMBus which is not part of I2C protocol. */
-    uint32_t ulBusFreq;         /**<! Bus frequency/baud rate */
+    uint32_t ulMasterTimeout; /**<! Master timeout value in msec, to relinquish the bus if slave does not respond. This is an extended feature of SMBus which is not part of I2C protocol. */
+    uint32_t ulBusFreq;       /**<! Bus frequency/baud rate */
 } IotI2CConfig_t;
 
 /**
  * @brief Ioctl request types.
  */
 typedef enum
 {
-    eI2CSendNoStopFlag,     /**<! Set flag to not send stop after transaction */
-                            /**<! Default is always stop for every transaction */
-                            /**<! Flag will auto reset to stop after one transaction if you set no stop */
-    eI2CSetSlaveAddr,       /**<! All the operations use this slave address after it is set. */
-    eI2CSetMasterConfig,    /**<! Sets the I2C bus frequency and timeout using the struct IotI2CConfig_t,
-                             *   default speed is Standard */
```

Comment:
Why remove this? Isn't this statement true?

Reply:
Not remove, just made it one line...

Reply:
Oh my bad

Reply:
Oh I realize what happened here. My git plugin is not displaying this PR properly